test/house_test.rb

```diff
@@ -0,0 +1,105 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def test_it_exists_and_has_attrs
+    house = House.new("$400000", "123 sugar lane")
+    assert_instance_of House, house
+    assert_equal "$400000", house.price
+    assert_equal "123 sugar lane", house.address
+    assert_equal [], house.rooms
+  end
+
+  def test_adding_rooms
+    house = House.new("$400000", "123 sugar lane")
```

Comment:
You can save yourself some time by utilizing a `setup` method to create your objects.